locale/es/LC_MESSAGES/django.po

```diff
@@ -2841,7 +2841,7 @@ msgstr "Algunas páginas de La Biblioteca de Wikipedia son navegables in tener q
 #: TWLight/users/templates/users/terms.html:291
 #, python-format
 msgid "In order to administer the Wikipedia Library program, we will retain the application data we collect from you for three years after your most recent login, unless you delete your account, as described below. You can log in and go to <a class=\"twl-links\" href=\"%(profile_url)s\">your profile page</a> in order to see the information associated with your account, and can download it in JSON format. You may access, update, restrict, or delete this information at any time, except for the information that is automatically retrieved from the projects. If you have any questions or concerns about the handling of your data, please contact <a class=\"twl-links\" href=\"mailto:wikipedialibrary@wikimedia.org?Subject=Wikipedia%%20Library%%20Card%%20account%%20deactivation\"> wikipedialibrary@wikimedia.org </a>."
-msgstr "Para administrar el programa de la Biblioteca de Wikipedia, conservaremos los datos de la aplicación que recopilamos de usted durante tres años después de su inicio de sesión más reciente, a menos que elimine su cuenta, como se describe a continuación. Puede iniciar sesión e ir a <a class=\"twl-links\" href=\"%(profile_url)s\">su página de perfil</a> para ver la información asociada con su cuenta y descargarla en formato JSON. Puede acceder, actualizar, restringir o eliminar esta información en cualquier momento, excepto la información que se recupera automáticamente de los proyectos. Si tiene alguna pregunta o inquietud sobre el manejo de sus datos, comuníquese con <a class=\"twl-links\" href=\"mailto:wikipedialibrary@wikimedia.org?Subject=Wikipedia%%20Library% <span class=\"notranslate\" translate=\"no\">%20Card% %20account% %20deactivation\"> wikipedialibrary@wikimedia.org</a> ."
+msgstr "Para administrar el programa de la Biblioteca de Wikipedia, conservaremos los datos de la aplicación que recopilamos de usted durante tres años después de su inicio de sesión más reciente, a menos que elimine su cuenta, como se describe a continuación. Puede iniciar sesión e ir a <a class=\"twl-links\" href=\"%(profile_url)s\">su página de perfil</a> para ver la información asociada con su cuenta y descargarla en formato JSON. Puede acceder, actualizar, restringir o eliminar esta información en cualquier momento, excepto la información que se recupera automáticamente de los proyectos. Si tiene alguna pregunta o inquietud sobre el manejo de sus datos, comuníquese con <a class=\"twl-links\" href=\"mailto:wikipedialibrary@wikimedia.org?Subject=Wikipedia%%20Library%%20Card%%20account%%20deactivation\"> wikipedialibrary@wikimedia.org</a>."
```

Comment:
So this was definitely our problem, but I'm not sure that we should be updating this directly.

Reply:
We should be fixing things like this in translatewiki. If this string doesn't match what's in translatewiki, then there is something weird going on.

Reply:
It worked on staging. I've already fixed the translation in TranslateWiki, so we can wait for Thursday's update.